save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -51,16 +84,51 @@ class UserSettingsOrganizationsMenuView : UserSettingsView() {
                                 }
                             }
                             div {
-                                className = ClassName("mr-3")
-                                val role = state.userInfo?.name?.let {
-                                    organizationDto.userRoles[it]
-                                } ?: Role.NONE
-                                +role.formattedName
+                                className = ClassName("col-5 align-self-right d-flex align-items-center justify-content-end")
+                                val role = state.userInfo?.name?.let { organizationDto.userRoles[it] } ?: Role.NONE
+                                div {
+                                    className = ClassName("mr-3")
+                                    role
+                                    +role.formattedName
+                                }
+                                if (role.isHigherOrEqualThan(Role.OWNER)) {
+                                    div {
+                                        button {
+                                            className = ClassName("btn mr-3")
+                                            fontAwesomeIcon(icon = faTrashAlt)
+                                            id = "remove-organization-${organizationDto.name}"
+                                            onClick = {
+                                                setDeleteOrganization(organizationDto)
+                                                setFlagDeleteOrganization(true)
+                                            }
+                                        }
+                                    }
+                                }
                             }
                         }
                     }
                 }
             }
         }
     }
+    private fun deleteOrganization(organizationDto: OrganizationDto) {
+        val headers = Headers().also {
+            it.set("Accept", "application/json")
+            it.set("Content-Type", "application/json")
+        }
+        lateinit var responseFromDeleteOrganization: Response
+        scope.launch {
+            responseFromDeleteOrganization =
+                    delete(
+                        "$apiUrl/organizations/${organizationDto.name}/delete",
+                        headers,
+                        body = undefined,
+                        loadingHandler = ::noopLoadingHandler,
+                    )
+        }.invokeOnCompletion {
+            if (responseFromDeleteOrganization.ok) {
+                window.location.reload()
```

Comment:
Not needed reload of page. You can delete Organization from _state.selfOrganizationDtos_